bin/s2nd.c

```diff
@@ -713,6 +735,43 @@ int main(int argc, char *const *argv)
         exit(1);
     }
 
+    if (conn_settings.session_ticket) {
+        if (s2n_config_set_session_tickets_onoff(config, 1) < 0) {
+            fprintf(stderr, "Error enabling session tickets: '%s'\n", s2n_strerror(s2n_errno, "EN"));
+            exit(1);
+        }
+
+        /* Key initialization */
+        uint8_t *st_key;
+        uint32_t st_key_length;
+
+        if (session_ticket_key_file_path) {
+            int fd = open(session_ticket_key_file_path, O_RDONLY);
+            if (fd < 0) {
+                fprintf(stderr, "Error opening session ticket key file: '%s'\n", strerror(errno));
```

Comment:
Here and below, might be cleaner to use error function:
```
error(1, errno, "Error opening session ticket key file");
```

Reply:
Done